documentation/announcing-az-module.md

```diff
@@ -85,92 +92,92 @@ PS C:\> Disable-AzureRmAlias -Scope LocalMachine
 ```
 
 ### Module Name Changes
-For scripts that import modules directly, or use ```#Requires``` statements to specify required modules, references to ```AzureRM.*``` will need to be changed to the appropriate ```Az``` module.  A current list of AzureRM modules and their Az equivalents are listed below.  Note that this list is subject to change as module names in Az are updated through November 2018.
+For scripts that import modules directly, or use ```#Requires``` statements to specify required modules, references to ```AzureRM.*``` will need to be changed to the appropriate ```Az``` module.  A current list of AzureRM modules and their Az equivalents are listed below.
 
 | AzureRM Module Name | Az Module Name|
 | -------------------------------------- | ------------------------------- |
 | AzureRM | Az |
 | Azure.AnalysisServices | Az.AnalysisServices |
-| Azure.Storage | Az.Storage |
+| Azure.Storage | **Az.Storage** |
```

Comment:
@markcowl nit: to be consistent with the other consolidated/changed modules below, `Azure.Storage` should be `**Azure.Storage**`